src/transformStateWithClones.js

```diff
@@ -7,7 +7,27 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let currentState = { ...state };
```

Comment:
[NAMING]: The name 'currentState' is a good choice for the object copy as it clearly indicates its purpose. Well done on following the naming convention.